website/markdown/docs/building-at-scale/caching.mdx

```diff
@@ -40,6 +40,13 @@ tuist cache warm
         'Cache the targets as .xcframeworks. It requires targets to be enabled for library distribution.',
       default: '`false`',
     },
+    {
+      long: '`--targets`',
+      short: '`-t`',
+      description:
```

Comment:
I'd also add a simple example to `Warming the cache`, something like:
If you want to cache only specific targets and their dependencies, you can run:
`tuist cache warm FrameworkA FrameworkB`